UPGRADE-0.12.md

```diff
@@ -0,0 +1,37 @@
+UPGRADE FROM 0.11 to 0.12
+=======================
+
+# Table of Contents
+
+- [Remove auto mapping configuration](#remove-auto-mapping-configuration)
+
+### Remove auto mapping configuration
+
+ * The AutoMapping configuration entry has been removed in favor of Symfony 4+ service configuration.
+
+  Upgrading:
+   - Delete old configuration.
+        ```diff
+        overblog_graphql:
+            definitions:
+        -        auto_mapping: ~
+        ```
+   - use Symfony 4+ service configuration to tag your types, resolvers or mutation.
+       ```yaml
+       # config/graphql.yaml
+       services:
+           _defaults:
+               autowire: true
+               public: true
+       
+           _instanceof:
+               GraphQL\Type\Definition\Type:
+                   tags: ['overblog_graphql.type']
+               Overblog\GraphQLBundle\Definition\Resolver\ResolverInterface:
+                   tags: ['overblog_graphql.resolver']
+               Overblog\GraphQLBundle\Definition\Resolver\MutationInterface:
+                   tags: ['overblog_graphql.mutation']
+       
+           App\GraphQL\:
```

Comment:
What's the idea of this one?

Reply:
@mcg-web ping here :D

Reply:
It is the entry where the DI will actually look for the classes. But I'm not sure I'm really understanding your question.

Reply:
I'm afraid this sample is implying people have a GraphQL namespace within their source, which we don't actually know. So I'm asking to know if there's something else related to this namespace which I might have missed

Reply:
I should modify to `resource: src/GraphQL` instead as this was the default dir of auto mapping. What do you think of that, is it more explicit?

Reply:
Maybe we don't even need this declaration, or is necessary for something?